### DISCUSSION TITLE
How Can I Download The Scratch Addons exe On A Chrome OS? [INVALID]

### DISCUSSION THREAD

**Discussion by JellyScript** [ScratchAddons/ScratchAddons#2273] [Help]
So, every time I download the asset, I change the file type to exe, hoping to download the extension. But every time it says it's for a different operating system. Help?
![Screenshot 2021-04-23 10 00 20 AM](https://user-images.githubusercontent.com/83035595/115905324-cbc93c00-a41a-11eb-972a-46ca59788fad.png)

**Comment by retronbv** [comment]
There is no Scratch Addons exe, where did you download this from?

**Comment by JellyScript** [comment]
LOL I changed the file type from .zip to .exe

**Reply by mxmou** [reply]
Scratch Addons isn't a Windows app so renaming it to .exe won't do anything. If you can't install extensions from the Chrome web store, you can't use it.

**Reply by GrahamSH-LLK** [reply]
> Scratch Addons isn't a Windows app so renaming it to .exe won't do anything. If you can't install extensions from the Chrome web store, you can't use it.

Not neccesarily, you can easily sideload it via `chrome://extensions`

**Comment by Illodrus** [comment]
There is no ScratchAddons `exe` file.
Changing a `zip` file to `exe` does not make it an executable.
ChromeOS cannot run `exe` files.

If you want to add the extension via the `zip` file, you need to unzip the file, then go to `chrome://extensions` and click the "Load Unpacked" button in the top right hand corner. Select the ScratchAddons folder you unzipped to upload it.

**Reply by GrahamSH-LLK** [reply]
This is factually incorrect, because we don't _have_ an `exe`. If you had read above, the user changed the zip to an `exe`.

**Reply by Illodrus** [reply]
> This is factually incorrect, because we don't _have_ an `exe`. If you had read above, the user changed the zip to an `exe`.

I changed my reply to more fully answer the question
